sendgrid/helpers/mail/mail.py

```diff
@@ -65,6 +66,13 @@ def __str__(self):
         """
         return str(self.get())
 
+    def __update_params(self, given_args, mail, arg_str):
```

Comment:
Using double-underscores is not usually what you want to do, wouldn't a single preceding underscore achieve the same goal here?